dateparser/languages/loader.py

```diff
@@ -34,10 +36,14 @@ def _load_data(self):
             data = get_data('data', 'languages.yaml')
         else:
             data = self.file.read()
-        data = load_yaml(data)
-        base_data = data.pop('base', {})
+        
+        if not self._raw_data:
```

Comment:
I wanted to use it as a static variable so we don't have to load data from yaml every time we create a new `LanguageDataLoader` instance. Fixed it for that.